styles/common.css

```
@@ -35,6 +36,37 @@ body {
     line-height: 26px;
 }
 
+.nav-mobile-icons {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-decoration: none;
+    font-family: DM Sans, Arial, Helvetica, sans-serif;
+    font-size: 1.25rem;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 26px;
+    margin-top: -130px;
+    background: #fdfafc;
```

É melhor usar `background-color` pra ser mais específico
```suggestion
    background-color: #fdfafc;
```

---

feito